src/Sulu/Bundle/CoreBundle/DependencyInjection/SuluCoreExtension.php

```diff
@@ -323,29 +323,20 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('doctrine.xml');
     }
 
-    /**
-     * @param $webspaceConfig
-     */
-    private function initWebspace($webspaceConfig, ContainerBuilder $container, Loader\XmlFileLoader $loader)
+    private function initWebspace(array $webspaceConfig, ContainerBuilder $container, XmlFileLoader $loader)
```

Comment:
we cannot add a `array` typehint, we need to use a phpdoc comment

```suggestion
    /**
    * @param array $webspaceConfig
    */
    private function initWebspace(array $webspaceConfig, ContainerBuilder $container, XmlFileLoader $loader)
```